sql/functions/pgbouncer_fdw_functions.sql

```diff
@@ -100,7 +101,50 @@ LOOP BEGIN
     INTO v_version_major, v_version_minor
     FROM @extschema@.pgbouncer_version_func(v_row.target_host);
   
-    IF v_version_major >= 1 AND v_version_minor >= 18 THEN 
+    IF v_version_major >= 1 AND v_version_minor >= 21 THEN 
```

Comment:
Hmm, technically this would fail for 2.0, since v_version_minor would fail the second part of the conditional in that case.  If this is a goal to support 2.0 on the off chance we can, then maybe this should be the final catch-all and move the other version checks ahead of it, otherwise you'd need to make this a compound statement along the lines if `IF (v_version_major = 1 AND v_version_minor >= 21) OR v_version_major > 1) THEN ...`, no?

Same applies at the other similar sites.

Reply:
Hmm true. I'll try and tackle that the next time an update is needed since we're trying to get this out soon. Thanks for pointing that out!

Reply:
capture this in an issue?

Reply:
Done! Thanks